Can't deploy slot in functions with linux runtime

### Plugin name and version
 - OS: Win/Mac
 - Plugin: Azure-functions-maven-plugin 1.9.1
### Plugin configuration in your `pom.xml`
```
<plugin>
    <groupId>com.microsoft.azure</groupId>
    <artifactId>azure-functions-maven-plugin</artifactId>
    <version>${azure.functions.maven.plugin.version}</version>
    <configuration>
       <appName>${functionAppName}</appName>
       <resourceGroup>functest</resourceGroup>
       <deploymentType>RUN_FROM_BLOB</deploymentType>
        <region>westus</region>
       <pricingTier>P1V2</pricingTier>
       <runtime>
          <os>linux</os>
          <javaVersion>8</javaVersion>
       </runtime>
       <deploymentSlot>
          <name>slot</name>
          <configurationSource>parent</configurationSource>
       </deploymentSlot>
       <appSettings>
          <property>
             <name>FUNCTIONS_EXTENSION_VERSION</name>
             <value>~3</value>
          </property>
       </appSettings>
    </configuration>
    <executions>
       <execution>
          <id>package-functions</id>
          <goals>
             <goal>package</goal>
          </goals>
       </execution>
    </executions>
 </plugin>
```
### Actual behavior
![image](https://user-images.githubusercontent.com/46642771/105477361-b68a2780-5cdc-11eb-8909-322a07ffb084.png)

```
[INFO] Set function worker runtime to java.
[INFO] The specified function slot does not exist. Creating a new slot...
[INFO] Successfully created the function slot: slot1.
[INFO] Updating the specified function slot...
[INFO] Successfully updated the function slot: slot1.
[INFO] Starting deployment...
[INFO] Trying to deploy artifact to slot1...
[INFO] Successfully deployed the artifact to
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:05 min
[INFO] Finished at: 2021-01-22T09:38:19+00:00
[INFO] Final Memory: 57M/761M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.microsoft.azure:azure-functions-maven-plugin:1.9.1:deploy (default-cli) on project xxxxxxx: com.microsoft.azure.management.appservice.implementation.FunctionDeploymentSlotImpl cannot be cast to com.microsoft.azure.management.appservice.FunctionApp -> [Help 1]
```

--- a/azure-tools-common/src/main/java/com/microsoft/azure/common/function/handlers/artifact/RunFromBlobArtifactHandlerImpl.java
+++ b/azure-tools-common/src/main/java/com/microsoft/azure/common/function/handlers/artifact/RunFromBlobArtifactHandlerImpl.java
@@ -52,7 +52,6 @@ public class RunFromBlobArtifactHandlerImpl extends ArtifactHandlerBase {
         final CloudBlockBlob blob = deployArtifactToAzureStorage(deployTarget, zipPackage, storageAccount);
         final String sasToken = AzureStorageHelper.getSASToken(blob, Period.ofYears(SAS_EXPIRE_DATE_BY_YEAR));
         FunctionArtifactHelper.updateAppSetting(deployTarget, APP_SETTING_WEBSITE_RUN_FROM_PACKAGE, sasToken);
-        ((FunctionApp) deployTarget.getApp()).syncTriggers();
     }
 
     private CloudBlockBlob deployArtifactToAzureStorage(DeployTarget deployTarget, File zipPackage, CloudStorageAccount storageAccount)

